rts/lib/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ INCLUDE_DIRECTORIES(${CMAKE_SOURCE_DIR}/rts)
 
 ADD_SUBDIRECTORY(lua)
 ADD_SUBDIRECTORY(luasocket)
+ADD_SUBDIRECTORY(meshoptimizer)
```

Comment:
side node: some additions in rts/lib seems to not contain the "LICENSE" files, i.e. https://github.com/spring/spring/blob/d4eefe88a865d41f4f25f3db072d1cad751728a7/rts/lib/xsimd/xsimd.hpp

when possible IMHO it would be better to use git submodules: this should make it easier to keep dependencies up to date.

thats maybe something which should be solved when we have a conclusion on about develop / transition branch changes + the roadmap.